.github/workflows/test.yml

```
@@ -11,10 +11,10 @@ permissions: {}
 jobs:
   test:
     env:
-      REQUIRE_ASPELL: true
+      REQUIRE_ASPELL: ${{ startsWith(matrix.os, 'ubuntu') }
```

actionlint (1.7.4)

got unexpected EOF while lexing end marker }}, expecting '}'

(expression)